src/m365/pp/commands/card/card-list.spec.ts

```diff
@@ -0,0 +1,214 @@
+      "@odata.context": "https://contoso-dev.api.crm4.dynamics.com/api/data/v9.1/$metadata#cards(owninguser(azureactivedirectoryobjectid,fullname))",
+      "value": [
+        {
+          "solutionid": "fd140aae-4df4-11dd-bd17-0019b9312238",
+          "modifiedon": "2022-10-11T08:52:12Z",
+          "_owninguser_value": "7d48edd3-69fd-ec11-82e5-000d3ab87733",
+          "overriddencreatedon": null,
+          "ismanaged": false,
+          "schemaversion": null,
+          "tags": null,
+          "importsequencenumber": null,
+          "componentidunique": "d7c1acb5-37a4-4873-b24e-34b18c15c6a5",
+          "_modifiedonbehalfby_value": null,
+          "componentstate": 0,
+          "statecode": 0,
+          "name": "DummyCard",
+          "versionnumber": 3044006,
+          "utcconversiontimezonecode": null,
+          "cardid": "69703efe-4149-ed11-bba2-000d3adf7537",
+          "publishdate": null,
+          "_createdonbehalfby_value": null,
+          "_modifiedby_value": "7d48edd3-69fd-ec11-82e5-000d3ab87733",
+          "createdon": "2022-10-11T08:52:12Z",
+          "overwritetime": "1900-01-01T00:00:00Z",
+          "_owningbusinessunit_value": "2199f44c-195b-ec11-8f8f-000d3adca49c",
+          "hiddentags": null,
+          "description": " ",
+          "appdefinition": "{\"screens\":{\"main\":{\"template\":{\"type\":\"AdaptiveCard\",\"body\":[{\"type\":\"TextBlock\",\"size\":\"Medium\",\"weight\":\"bolder\",\"text\":\"Your card title goes here\"},{\"type\":\"TextBlock\",\"text\":\"Add and remove element to customize your new card.\",\"wrap\":true}],\"actions\":[],\"$schema\":\"http://adaptivecards.io/schemas/1.4.0/adaptive-card.json\",\"version\":\"1.4\"},\"verbs\":{\"submit\":\"echo\"}}},\"sampleData\":{\"main\":{}},\"connections\":{},\"variables\":{},\"flows\":{}}",
+          "statuscode": 1,
+          "remixsourceid": null,
+          "sizes": null,
+          "_owningteam_value": null,
+          "coowners": null,
+          "_createdby_value": "7d48edd3-69fd-ec11-82e5-000d3ab87733",
+          "_ownerid_value": "7d48edd3-69fd-ec11-82e5-000d3ab87733",
+          "publishsourceid": null,
+          "timezoneruleversionnumber": null,
+          "iscustomizable": {
+            "Value": true,
+            "CanBeChanged": true,
+            "ManagedPropertyLogicalName": "iscustomizableanddeletable"
+          },
+          "owninguser": {
+            "azureactivedirectoryobjectid": "88e85b64-e687-4e0b-bbf4-f42f5f8e574c",
+            "fullname": "Contoso Admin",
+            "systemuserid": "7d48edd3-69fd-ec11-82e5-000d3ab87733",
+            "ownerid": "7d48edd3-69fd-ec11-82e5-000d3ab87733"
+          }
+        }
+      ]
+    }
+  };
+  //#endregion
+
+  let log: string[];
+  let logger: Logger;
+  let loggerLogSpy: sinon.SinonSpy;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(appInsights, 'trackEvent').callsFake(() => { });
+    auth.service.connected = true;
```

Comment:
Could you add a `getProcessName` stub like we do here?
https://github.com/pnp/cli-microsoft365/pull/3769/files